documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,103 @@
+> Release Issue Template
+
+# Lotus X.Y.Z Release
+
+We're happy to announce Lotus X.Y.Z...
+
+## 🗺 What's left for release
+
+<List of items with PRs and/or Issues to be considered for this release>
+
+## 🚢 Estimated shipping date
+
+<Date this release will ship on if everything goes to plan (week beginning...)>
+
+## 🔦 Highlights
+
+< top highlights for this release notes >
+
+## Changelog
+
+< changelog generated by bin/mkreleaselog >
```

Comment:
I'd actually consider not including this in the issue. It tends to make it a bit unwieldy.

Reply:
But up to you.

Reply:
sgtm